src/shared/utils/mixpanel/mixpanel.utils.ts

```diff
@@ -27,40 +27,98 @@ export const trackAppletSave = ({
 }) => {
   if (!applet) return;
 
-  const itemTypes: ItemResponseType[] = [];
+  let itemCount = 0;
+  let phraseBuilderItemCount = 0;
+  let itemsIncludedInPhraseBuilders = 0;
+  const phraseBuilderFieldCounts = {
+    itemResponse: 0,
+    sentence: 0,
+    lineBreak: 0,
+  };
   let autoAssignedActivityCount = 0;
   let manualAssignedActivityCount = 0;
-  for (const activity of applet.activities ?? []) {
-    itemTypes.push(...(activity.items ?? []).map((item) => item.responseType));
+  const itemTypes: ItemResponseType[] = [];
+
+  for (const activity of applet.activities) {
     if (activity.autoAssign) {
       autoAssignedActivityCount++;
     } else {
       manualAssignedActivityCount++;
     }
+
+    const visibleItemNames = new Set<string>();
+    const referencedItemNames = new Set<string>();
```

Comment:
I think this is a remnant of the previous implementation. It's shadowed in the inner loop and doesn't end up doing anything, so we can remove it

Reply:
Ah yes, the result of my not doing a careful enough rebase!